brayns/common/transferFunction/TransferFunction.cpp

```diff
@@ -0,0 +1,182 @@
+#include "TransferFunction.h"
+
+#include <brayns/common/log.h>
+
+namespace
+{
+    const std::string TF_RED_STRING = "red";
+    const std::string TF_GREEN_STRING = "green";
+    const std::string TF_BLUE_STRING = "blue";
+    const std::string TF_ALPHA_STRING = "alpha";
+    const std::string TF_EMISSION_STRING = "emission";
+}
+
+namespace brayns
+{
+
+TransferFunction::TransferFunction()
+{
+}
+
+void TransferFunction::clear()
+{
+    _controlPoints.clear();
+    _diffuseColors.clear();
+    _emissionIntensities.clear();
+}
+
+Vector2fs& TransferFunction::getControlPoints( const TransferFunctionAttribute attribute )
+{
+    return _controlPoints[ attribute ];
+}
+
+void TransferFunction::resample( const size_t sampleSize )
```

Comment:
Some of this function should be in the lexis for common usage. Since there will be the threshold ( on default it is max and min of the data )  they shall be given as parameters as well.